src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
+                    "description": "Die reine Existenz eines (digitalen) Dokuments hat keine Nachweis-/Beweiskraft. Vielmehr <b>muss</b> jedes Dokument/Zertifikat auch auf die <a href='#glossary_validity'>Gültigkeit und Validität</a> überprüft werden. <br/><br/> Diese Prüfung erledigt eine spezielle App (in Deutschland die <a href='#glossary_covpasscheck'>CovPassCheck-App</a>), die Prüf-App. <br/><br/> Die einzelnen Länder haben möglicherweise unterschiedliche Implementierungen der Prüf-App. Aber alle Prüf-Apps nutzen dieselben – auf einem europäischen Gateway konsolidieren – Schlüssel der teilnehmenden Länder sowie die hinterlegten <a href='#glossary_business_rules'>Business Rules</a>. Daher können alle Prüf-Apps künftig auch alle EU-Zertifikate validieren, sofern sie der Spezifikation des europäischen eHealth Network folgen."
+                },
+                {
+                    "term": "Pseudonymität",
+                    "anchor": "pseudonymity",
+                    "description": "Eine Maßnahme des Datenschutzes, um Daten entweder gar nicht oder nur sehr schwer, bestimmten Personen zuordnen zu lassen. Wie die Anonymisierung dient die Pseudonymisierung der Verhinderung eines Rückschlusses auf eine Person bzw. auf die schutzwürdigen Daten einer Person. Bei der Pseudonymisierung geschieht dies durch das Trennen der Merkmale von den anderen personenbezogenen Daten. Damit unterscheidet es sich von der Anonymisierung, bei welcher die identifizierenden Merkmale gelöscht werden."
+                }
+            ],
+            "Q": [
+                {
+                    "term": "Quarantäne",
+                    "anchor": "quarantine",
+                    "description": "Eine Schutzmaßnahme die dazu beiträgt die Verbreitung ansteckender Krankheiten mithilfe von menschlicher Isolation einzudämmen."
+                }
+            ],
+            "R": [
+                {
+                    "term": "RAT",
+                    "anchor": "rat",
+                    "description": "Abkürzung für Antigen-Schnelltest (Rapid Antigen Test). Siehe <a href='#glossary_rapid_antigen_test'>Antigen-Schnelltest</a>"
+                },
+                {
+                    "term": "Release",
+                    "anchor": "release",
+                    "description": "Version der Corona-Warn-App"
+                },
+                {
+                    "term": "Risikobegegnung",
+                    "anchor": "risk_encounter",
+                    "description": "aufgezeichnete Begegnung mit einer Person, die einen selbst möglicherweise aufgrund von Dauer und Abstand zur Infektion geführt haben könnte"

```

Comment:
Also here, missing full stop at end of sentence.

Reply:
Will be changed in the next commit, thanks!